workflows/iuf/operations/managed-nodes/managed-nodes-rollout.yaml

```diff
@@ -203,17 +203,17 @@ spec:
 
               rm "${tmpfile}"
               if [ "$session_templates_string" == "" ]; then
-                  echo "ERROR: Failed to parse session_templates from prepare_images_output"
+                  echo "ERROR Failed to parse session templates from 'prepare_images' stage"
```

Comment:
"The managed-nodes-rollout stage relies on the BOS session templates created in the prepare-images stage. Run the prepare-images stage and try again"